torch/csrc/autograd/FunctionsManual.h

```diff
@@ -220,6 +220,10 @@ at::Tensor unsqueeze_to(
     const at::Tensor& self,
     int64_t dim,
     c10::SymIntArrayRef sym_sizes);
+at::Tensor unsqueeze_to(
```

Comment:
The one above could be removed no?

Reply:
Both `squeeze.dim` and `squeeze.dims` call `unsqueeze_to` in their backwards, so we do use both overloads. I've rewritten the `int` version to just call the `int[]` version to reduce duplication though.